### PR TITLE
Update Broken Links

### PR DESCRIPTION
**Pull request by mikeamoss**
Fix broken links from legacy Technet link redirects

### CODE REVIEW

File: WindowsServerDocs/networking/technologies/dhcp/dhcp-deploy-wps.md [hunk 1]
```diff
@@ -316,8 +316,8 @@ Set-DnsClientServerAddress -InterfaceAlias "Ethernet" -ServerAddresses 10.0.0.2
 
 For more information about these commands, see the following topics.
 
-- [New-NetIPAddress](https://technet.microsoft.com/itpro/powershell/windows/tcpip/new-netipaddress)
-- [Set-DnsClientServerAddress](https://technet.microsoft.com/itpro/powershell/windows/dns-client/set-dnsclientserveraddress)
+- [New-NetIPAddress](https://docs.microsoft.com/en-us/powershell/module/nettcpip/New-NetIPAddress)
```
Comment (ItzLevvie) [inline WindowsServerDocs/networking/technologies/dhcp/dhcp-deploy-wps.md]:
```suggestion
- [New-NetIPAddress](https://docs.microsoft.com/powershell/module/nettcpip/New-NetIPAddress)
```

File: WindowsServerDocs/networking/technologies/dhcp/dhcp-deploy-wps.md [hunk 2]
```diff
@@ -316,8 +316,8 @@ Set-DnsClientServerAddress -InterfaceAlias "Ethernet" -ServerAddresses 10.0.0.2
...
+- [Set-DnsClientServerAddress](https://docs.microsoft.com/en-us/powershell/module/dnsclient/Set-DnsClientServerAddress)
```
Comment (ItzLevvie) [inline WindowsServerDocs/networking/technologies/dhcp/dhcp-deploy-wps.md]:
```suggestion
- [Set-DnsClientServerAddress](https://docs.microsoft.com/powershell/module/dnsclient/Set-DnsClientServerAddress)
```

File: WindowsServerDocs/networking/technologies/dhcp/dhcp-deploy-wps.md [hunk 3]
```diff
@@ -361,7 +361,7 @@ Install-WindowsFeature DHCP -IncludeManagementTools
 
 For more information about this command, see the following topic.
 
-- [Install-WindowsFeature](https://technet.microsoft.com/itpro/powershell/windows/server-manager/install-windowsfeature)
+- [Install-WindowsFeature](https://docs.microsoft.com/en-us/powershell/module/Microsoft.Windows.ServerManager.Migration/Install-WindowsFeature)
```
Comment (ItzLevvie) [inline WindowsServerDocs/networking/technologies/dhcp/dhcp-deploy-wps.md]:
```suggestion
- [Install-WindowsFeature](https://docs.microsoft.com/powershell/module/Microsoft.Windows.ServerManager.Migration/Install-WindowsFeature)
```

File: WindowsServerDocs/networking/technologies/dhcp/dhcp-deploy-wps.md [hunk 4]
```diff
@@ -416,8 +416,8 @@ IPAddress	DnsName
 
 For more information about these commands, see the following topics.
 
-- [Add-DhcpServerInDC](https://technet.microsoft.com/itpro/powershell/windows/dhcp-server/add-dhcpserverindc)
-- [Get-DhcpServerInDC](https://technet.microsoft.com/itpro/powershell/windows/dhcp-server/get-dhcpserverindc)
+- [Add-DhcpServerInDC](https://docs.microsoft.com/en-us/powershell/module/dhcpserver/add-dhcpserverindc?view=win10-ps)
```
Comment (ItzLevvie) [inline WindowsServerDocs/networking/technologies/dhcp/dhcp-deploy-wps.md]:
```suggestion
- [Add-DhcpServerInDC](https://docs.microsoft.com/powershell/module/dhcpserver/add-dhcpserverindc?view=win10-ps)
```

File: WindowsServerDocs/networking/technologies/dhcp/dhcp-deploy-wps.md [hunk 5]
```diff
@@ -416,8 +416,8 @@ IPAddress	DnsName
...
+- [Get-DhcpServerInDC](https://docs.microsoft.com/en-us/powershell/module/dhcpserver/get-dhcpserverindc)
```
Comment (ItzLevvie) [inline WindowsServerDocs/networking/technologies/dhcp/dhcp-deploy-wps.md]:
```suggestion
- [Get-DhcpServerInDC](https://docs.microsoft.com/powershell/module/dhcpserver/get-dhcpserverindc)
```

File: WindowsServerDocs/networking/technologies/dhcp/dhcp-deploy-wps.md [hunk 6]
```diff
@@ -450,8 +450,8 @@ Set-DhcpServerDnsCredential -Credential $Credential -ComputerName "DHCP1.corp.co
 
 For more information about these commands, see the following topics.
 
-- [Set-DhcpServerv4DnsSetting](https://technet.microsoft.com/itpro/powershell/windows/dhcp-server/set-dhcpserverv4dnssetting)
-- [Set-DhcpServerDnsCredential](https://technet.microsoft.com/itpro/powershell/windows/dhcp-server/set-dhcpserverdnscredential)
+- [Set-DhcpServerv4DnsSetting](https://docs.microsoft.com/en-us/powershell/module/dhcpserver/set-dhcpserverv4dnssetting)
+- [Set-DhcpServerDnsCredential](https://docs.microsoft.com/en-us/powershell/module/dhcpserver/set-dhcpserverdnscredential)
```
Comment (ItzLevvie) [inline WindowsServerDocs/networking/technologies/dhcp/dhcp-deploy-wps.md]:
```suggestion
- [Set-DhcpServerDnsCredential](https://docs.microsoft.com/powershell/module/dhcpserver/set-dhcpserverdnscredential)
```

File: WindowsServerDocs/networking/technologies/dhcp/dhcp-deploy-wps.md [hunk 7]
```diff
@@ -450,8 +450,8 @@ Set-DhcpServerDnsCredential -Credential $Credential -ComputerName "DHCP1.corp.co
 
 For more information about these commands, see the following topics.
 
-- [Set-DhcpServerv4DnsSetting](https://technet.microsoft.com/itpro/powershell/windows/dhcp-server/set-dhcpserverv4dnssetting)
-- [Set-DhcpServerDnsCredential](https://technet.microsoft.com/itpro/powershell/windows/dhcp-server/set-dhcpserverdnscredential)
+- [Set-DhcpServerv4DnsSetting](https://docs.microsoft.com/en-us/powershell/module/dhcpserver/set-dhcpserverv4dnssetting)
```
Comment (ItzLevvie) [inline WindowsServerDocs/networking/technologies/dhcp/dhcp-deploy-wps.md]:
```suggestion
- [Set-DhcpServerv4DnsSetting](https://docs.microsoft.com/powershell/module/dhcpserver/set-dhcpserverv4dnssetting)
```

File: WindowsServerDocs/networking/technologies/dhcp/dhcp-deploy-wps.md [hunk 8]
```diff
@@ -466,9 +466,9 @@ Set-DhcpServerv4OptionValue -DnsDomain corp.contoso.com -DnsServer 10.0.0.2
 
 For more information about these commands, see the following topics.
 
-- [Add-DhcpServerv4Scope](https://technet.microsoft.com/itpro/powershell/windows/dhcp-server/add-dhcpserverv4scope)
-- [Add-DhcpServerv4ExclusionRange](https://technet.microsoft.com/itpro/powershell/windows/dhcp-server/add-dhcpserverv4exclusionrange)
-- [Set-DhcpServerv4OptionValue](https://technet.microsoft.com/itpro/powershell/windows/dhcp-server/set-dhcpserverv4optionvalue)
+- [Add-DhcpServerv4Scope](https://docs.microsoft.com/en-us/powershell/module/dhcpserver/add-dhcpserverv4scope)
+- [Add-DhcpServerv4ExclusionRange](https://docs.microsoft.com/en-us/powershell/module/dhcpserver/add-dhcpserverv4exclusionrange)
+- [Set-DhcpServerv4OptionValue](https://docs.microsoft.com/en-us/powershell/module/dhcpserver/set-dhcpserverv4optionvalue)
```
Comment (ItzLevvie) [inline WindowsServerDocs/networking/technologies/dhcp/dhcp-deploy-wps.md]:
```suggestion
- [Set-DhcpServerv4OptionValue](https://docs.microsoft.com/powershell/module/dhcpserver/set-dhcpserverv4optionvalue)
```

File: WindowsServerDocs/networking/technologies/dhcp/dhcp-deploy-wps.md [hunk 9]
```diff
@@ -466,9 +466,9 @@ Set-DhcpServerv4OptionValue -DnsDomain corp.contoso.com -DnsServer 10.0.0.2
 
 For more information about these commands, see the following topics.
 
-- [Add-DhcpServerv4Scope](https://technet.microsoft.com/itpro/powershell/windows/dhcp-server/add-dhcpserverv4scope)
-- [Add-DhcpServerv4ExclusionRange](https://technet.microsoft.com/itpro/powershell/windows/dhcp-server/add-dhcpserverv4exclusionrange)
-- [Set-DhcpServerv4OptionValue](https://technet.microsoft.com/itpro/powershell/windows/dhcp-server/set-dhcpserverv4optionvalue)
+- [Add-DhcpServerv4Scope](https://docs.microsoft.com/en-us/powershell/module/dhcpserver/add-dhcpserverv4scope)
+- [Add-DhcpServerv4ExclusionRange](https://docs.microsoft.com/en-us/powershell/module/dhcpserver/add-dhcpserverv4exclusionrange)
```
Comment (ItzLevvie) [inline WindowsServerDocs/networking/technologies/dhcp/dhcp-deploy-wps.md]:
```suggestion
- [Add-DhcpServerv4ExclusionRange](https://docs.microsoft.com/powershell/module/dhcpserver/add-dhcpserverv4exclusionrange)
```

File: WindowsServerDocs/networking/technologies/dhcp/dhcp-deploy-wps.md [hunk 10]
```diff
@@ -466,9 +466,9 @@ Set-DhcpServerv4OptionValue -DnsDomain corp.contoso.com -DnsServer 10.0.0.2
 
 For more information about these commands, see the following topics.
 
-- [Add-DhcpServerv4Scope](https://technet.microsoft.com/itpro/powershell/windows/dhcp-server/add-dhcpserverv4scope)
-- [Add-DhcpServerv4ExclusionRange](https://technet.microsoft.com/itpro/powershell/windows/dhcp-server/add-dhcpserverv4exclusionrange)
-- [Set-DhcpServerv4OptionValue](https://technet.microsoft.com/itpro/powershell/windows/dhcp-server/set-dhcpserverv4optionvalue)
+- [Add-DhcpServerv4Scope](https://docs.microsoft.com/en-us/powershell/module/dhcpserver/add-dhcpserverv4scope)
```
Comment (ItzLevvie) [inline WindowsServerDocs/networking/technologies/dhcp/dhcp-deploy-wps.md]:
```suggestion
- [Add-DhcpServerv4Scope](https://docs.microsoft.com/powershell/module/dhcpserver/add-dhcpserverv4scope)
```

File: WindowsServerDocs/networking/technologies/dhcp/dhcp-deploy-wps.md [hunk 11]
```diff
@@ -503,14 +503,14 @@ The following reference provides command descriptions and syntax for all DHCP Se
 >[!NOTE]
 >You can not use Windows Server 2016 commands in Windows Server 2012 R2.
 
-- [DhcpServer Module](https://technet.microsoft.com/itpro/powershell/windows/dhcp-server/index)
+- [DhcpServer Module](https://docs.microsoft.com/en-us/powershell/module/dhcpserver/?view=win10-ps)
```
Comment (ItzLevvie) [inline WindowsServerDocs/networking/technologies/dhcp/dhcp-deploy-wps.md]:
```suggestion
- [DhcpServer Module](https://docs.microsoft.com/powershell/module/dhcpserver/?view=win10-ps)
```

File: WindowsServerDocs/networking/technologies/dhcp/dhcp-deploy-wps.md [hunk 12]
```diff
@@ -493,7 +493,7 @@ If the client does not receive an IP address from your DHCP server, perform the
 
 1. Ensure that the Ethernet cable is plugged into both the computer and the Ethernet switch, hub,  or router.
 2. If you plugged the client computer into a network segment that is separated from the DHCP server by a router, ensure that the router is configured to forward DHCP messages.
-3. Ensure that the DHCP server is authorized in Active Directory by running the following command to retrieve the list of authorized DHCP servers from Active Directory. [Get-DhcpServerInDC](https://technet.microsoft.com/itpro/powershell/windows/dhcp-server/get-dhcpserverindc).
+3. Ensure that the DHCP server is authorized in Active Directory by running the following command to retrieve the list of authorized DHCP servers from Active Directory. [Get-DhcpServerInDC](https://docs.microsoft.com/en-us/powershell/module/dhcpserver/get-dhcpserverindc).
```
Comment (ItzLevvie) [inline WindowsServerDocs/networking/technologies/dhcp/dhcp-deploy-wps.md]:
```suggestion
3. Ensure that the DHCP server is authorized in Active Directory by running the following command to retrieve the list of authorized DHCP servers from Active Directory. [Get-DhcpServerInDC](https://docs.microsoft.com/powershell/module/dhcpserver/get-dhcpserverindc).
```

File: WindowsServerDocs/networking/technologies/dhcp/dhcp-deploy-wps.md [hunk 13]
```diff
@@ -503,14 +503,14 @@ The following reference provides command descriptions and syntax for all DHCP Se
...
-- [DhcpServer Module](https://technet.microsoft.com/itpro/powershell/windows/dhcp-server/index)
+- [DhcpServer Module](https://docs.microsoft.com/en-us/powershell/module/dhcpserver/?view=win10-ps)
 
 The following reference provides command descriptions and syntax for all DHCP Server Windows PowerShell commands for Windows Server 2012 R2. The topic lists commands in alphabetical order based on the verb at the beginning of the commands, such as **Get** or **Set**.
 
 >[!NOTE]
 >You can use Windows Server 2012 R2 commands in Windows Server 2016.
 
-- [DHCP Server Cmdlets in Windows PowerShell](https://technet.microsoft.com/library/jj590751.aspx)
+- [DHCP Server Cmdlets in Windows PowerShell](https://docs.microsoft.com/en-us/powershell/module/dhcpserver/?view=winserver2012r2-ps)
```
Comment (ItzLevvie) [inline WindowsServerDocs/networking/technologies/dhcp/dhcp-deploy-wps.md]:
```suggestion
- [DHCP Server Cmdlets in Windows PowerShell](https://docs.microsoft.com/powershell/module/dhcpserver/?view=winserver2012r2-ps)
```

**Comment by illfated** [comment]
I concur with the suggested changes. Even if the pages are only available in English (en-us) at this time, it is usually a good idea to keep the links language agnostic, just in case some of them are (or will become) available in other languages.

**Comment by eross-msft** [comment]
@ItzLevvie @mikeamoss  I believe since you added the change, you must use the **Commit suggestion** button for each of your suggestions. I can't merge anything until your suggestions are part of the actual PR. This must be done in the next 30 days or it will become necessary to close this PR.

@mikeamoss  You must also fix the conflict while you're updating the commits.

**Comment by illfated** [comment]
In case @mikeamoss does not return to GitHub to see these messages for him, I am willing to open a new PR for each of his abandoned ones in this repository.

**Comment by eross-msft** [comment]
@illfated  Closing this item. Please open a new PR so we can go ahead and accept these changes.

**Comment by illfated** [comment]
Already taken care of. See list of commits & PR directly above this comment.

**Pull request closed without merging**